canonical-web-and-design/ubuntu.com
# Account users/user row

## Done

- Add User Row with 'regular', 'editing' and 'disabled' state
  - Actions do not interact with any backend, edit, deletion functionality etc. will be implemented separately
- add `@testing-library/jest-dom` (already recommended by @hatched here: https://docs.google.com/document/d/1ysVlPByzVY-NBUjGic1WupHojUl30P6cf3Xe-OVRRPg/edit#heading=h.qqiuv5t530vo)
- refactor `setupTests` to TS and add to `tsconfig.json` to enable TS suggestions.

### regular:
![image](https://user-images.githubusercontent.com/7452681/132477250-32cbb3e2-9f8d-498d-bb42-da8328e8171b.png)
### edit:
![image](https://user-images.githubusercontent.com/7452681/132477287-30e44b66-9790-4940-a641-fae362de43be.png)
### disabled:
![image](https://user-images.githubusercontent.com/7452681/132477318-8ac94c67-cc13-40f1-b935-d829fb69fb62.png)


## QA

- Log in
- Go to https://ubuntu-com-10327.demos.haus/advantage/users
- confirm the state of each row can be toggled via edit/cancel/save buttons


## Issue / Card

https://github.com/canonical-web-and-design/commercial-squad/issues/199

## Screenshots
![image](https://user-images.githubusercontent.com/7452681/132476889-03499fa1-8ed9-416f-a051-9acc04615f19.png)
![image](https://user-images.githubusercontent.com/7452681/132476919-633deecb-e852-477a-8e97-beeded54b3c1.png)
